sm3/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "sm3"
```

Comment:
It looks like there is already a published `sm3` crate:

https://crates.io/crates/sm3

The repo/source code is here:

https://github.com/andelf/rust-sca/tree/master/sm3

Perhaps we could ask the author about upstreaming it.

Reply:
Putting these crates on RustCrypto would be a better choice, I will try to contact the original author. Thanks.

Reply:
I've invited @newpavlov as an owner of `sm3` on crates.io.

Feel free to overwrite it with RustCrypto crates. 😁

Also, I've invited @uudiin to `rust-sca` repo. You can update any READMEs and links.

Reply:
Wow, awesome, thank you!